http/transport.go

```diff
@@ -557,6 +576,63 @@ func getInstanceID(req *http.Request) flux.InstanceID {
 	return flux.InstanceID(s)
 }
 
+func errorResponse(w http.ResponseWriter, r *http.Request, apiError error) {
+	var outErr *flux.BaseError
+	var code int
+	err := errors.Cause(apiError)
+	switch err := err.(type) {
+	case flux.Missing:
+		code = http.StatusNotFound
+		outErr = err.BaseError
+	case flux.UserConfigProblem:
+		code = http.StatusUnprocessableEntity
+		outErr = err.BaseError
+	case flux.ServerException:
+		code = http.StatusInternalServerError
+		outErr = err.BaseError
+	default:
+		code = http.StatusInternalServerError
+		outErr = flux.CoverAllError(apiError)
+	}
+
+	writeError(w, r, code, outErr)
+}
```

Comment:
When doing this elsewhere, I've added a `http/render` package with `render.Error(w, r, err)`.

Aside: and generally a `render.Json(w, r, statusCode, obj)` as well